packages/react/accordion/src/Accordion.stories.tsx

```diff
@@ -11,74 +11,235 @@ import {
+          </AccordionPanel>
+        </AccordionItem>
+        <AccordionItem className={itemClass} value="four">
+          <AccordionHeader className={headerClass}>
+            <AccordionButton className={buttonClass}>Four</AccordionButton>
+          </AccordionHeader>
+          <AccordionPanel className={panelClass}>
+            Odio placerat <a href="#">quisque</a> sapien sagittis non sociis ligula penatibus
+            dignissim vitae, enim vulputate nullam semper potenti etiam volutpat libero.
+            <button>Cool</button>
+          </AccordionPanel>
+        </AccordionItem>
+      </Accordion>
+
+      <h1>Controlled</h1>
+      <Accordion type="single" value={valueOne} onValueChange={setValueOne} className={rootClass}>
+        <AccordionItem className={itemClass} value="one">
+          <AccordionHeader className={headerClass}>
+            <AccordionButton className={buttonClass}>One</AccordionButton>
+          </AccordionHeader>
+          <AccordionPanel className={panelClass}>
+            Per erat orci nostra luctus sociosqu mus risus penatibus, duis elit vulputate viverra
+            integer ullamcorper congue curabitur sociis, nisi malesuada scelerisque quam suscipit
+            habitant sed.
+          </AccordionPanel>
+        </AccordionItem>
+        <AccordionItem className={itemClass} value="two">
+          <AccordionHeader className={headerClass}>
+            <AccordionButton className={buttonClass}>Two</AccordionButton>
+          </AccordionHeader>
+          <AccordionPanel className={panelClass}>
+            Cursus sed mattis commodo fermentum conubia ipsum pulvinar sagittis, diam eget bibendum
+            porta nascetur ac dictum, leo tellus dis integer platea ultrices mi.
+          </AccordionPanel>
+        </AccordionItem>
+        <AccordionItem className={itemClass} value="three" disabled>
+          <AccordionHeader className={headerClass}>
+            <AccordionButton className={buttonClass}>Three (disabled)</AccordionButton>
+          </AccordionHeader>
+          <AccordionPanel className={panelClass}>
+            Sociis hac sapien turpis conubia sagittis justo dui, inceptos penatibus feugiat
+            himenaeos euismod magna, nec tempor pulvinar eu etiam mattis.
+          </AccordionPanel>
+        </AccordionItem>
+        <AccordionItem className={itemClass} value="four">
+          <AccordionHeader className={headerClass}>
+            <AccordionButton className={buttonClass}>Four</AccordionButton>
+          </AccordionHeader>
+          <AccordionPanel className={panelClass}>
+            Odio placerat <a href="#">quisque</a> sapien sagittis non sociis ligula penatibus
+            dignissim vitae, enim vulputate nullam semper potenti etiam volutpat libero.
+            <button>Cool</button>
+          </AccordionPanel>
+        </AccordionItem>
+      </Accordion>
+
+      <h1>Controlled (without close)</h1>
```

Comment:
I got confused by this title as I was expected none to close, but I guess this is maybe more like 

```suggestion
      <h1>Controlled (at least one open)</h1>
```

?

Reply:
Ah yes, that is exactly what it is, thanks :+1: